TestFrame/Models/CountriesModels/CapitalInfoModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Text.Json.Serialization;
+
+namespace TestFrame.Models.CountriesModels
+{
+    public class CapitalInfoModel
+    {
+        [JsonPropertyName("latlang")]
+        public List<float> Latlang { get; set; }
```

Comment:
if nothing specific from definitions, use double. It will always fit a float nicely, not the other way around